tests/db_connector/test_connector.py

```diff
@@ -0,0 +1,73 @@
+import os
+import unittest
+
+from tiledb.cloud.sql.tiledb_connection import TileDBConnection
+
+
+class TileDBConnectionTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Using setUpClass to run query only once for this class
+        con = TileDBConnection()
+        cls.cur = con.cursor()
+        cls.cur.execute("SELECT * from `tiledb://TileDB-Inc/quickstart_dense`")
```

Comment:
One thing before submission: Could you add a parameter substitution to this test? That way we can verify that that part works. Maybe like `where rows > ?` and provide 2, or any other formulation you prefer.

Reply:
Could you explain this a bit more? You mean using it like this:
``` cls.cur.execute("SELECT * from `tiledb://TileDB-Inc/quickstart_dense` WHERE rows > ?", (2, 3, 4))```

and essentially run the tests as many times as the values in the list for the placeholder? And in each test adapt the `results` variable according to this value?

Reply:
Sorry I didn’t get to this before I was off for a week! I would say you really only need to include 1 parameter here (e.g. `execute("select * from uri where rows > ?", (0,))`). That way we can be sure that substituting in parameters works correctly.

Reply:
tests passed!